src/thread_task.rs

```diff
@@ -772,17 +751,11 @@ unsafe fn delayed_fg_task(c: *const Rav1dContext, ttd: *mut TaskThreadData) {
                 abort();
             }
         }
-        row = ::core::intrinsics::atomic_xadd_seqcst(
-            &mut *((*ttd).delayed_fg.progress).as_mut_ptr().offset(0) as *mut atomic_int,
-            1 as c_int,
-        );
+        row = (*ttd).delayed_fg.progress[0].fetch_add(1, Ordering::SeqCst);
         #[allow(unused_assignments)]
```

Comment:
Can you get rid of this and my comment now?

Reply:
It never needed to exist afaict. The compiler is right, this value is never read. I think at one time it was but the way the C code is structured now it is not used. I think this should be a separate PR though, just in case I'm wrong and it turns out to break something.

Reply:
It can't break anything; it's just disabling warnings.